api/src/main/java/org/openmrs/module/shr/cdahandler/api/CdaImportService.java

```diff
@@ -17,13 +17,7 @@
 import java.util.List;
 
 import org.marc.everest.rmim.uv.cdar2.pocd_mt000040uv.ClinicalDocument;
-import org.openmrs.Concept;
-import org.openmrs.ConceptReferenceTerm;
-import org.openmrs.ConceptSource;
-import org.openmrs.Obs;
-import org.openmrs.Order;
-import org.openmrs.Visit;
-import org.openmrs.activelist.ActiveListItem;
+import org.openmrs.*;
```

Comment:
Well they already have them like this in other places, so it's fine - but for the future we want to avoid them